docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -641,10 +651,11 @@ For 'non host vars' you can use the :ref:`vars lookup<vars_lookup>` plugin:
 
 .. _why_no_wheel:
 
-Why don't you ship in X format?
-+++++++++++++++++++++++++++++++
+Why don't you ship ansible in wheel format (or other format) ?
```

Comment:
Added back Wheel (as example) and added a more generic case after ("X format" is not obvious what is the topic